### DISCUSSION TITLE
[TESTING] I need some knowledge regarding practice

### DISCUSSION THREAD

**Discussion by wplatform** [azerothcore/azerothcore-wotlk#3951] [General]
I know the azerothcore needs more memory than trinitycore, but for a playable server, I have to migrate  from trinitycore to azerothcore. So I want more knowledge regarding the practice.
1. Is the project support localization for Chinese? 
2. Can the Portal Master(https://rochet2.github.io/Portal-Master.html) works on the azerothcore.
3. How to setting script instance drop items.
4. Are there some azerothcore repack to refer to setting a best interesting blizzardlike server.
5. How to implement if the player reaches 80 level. send some items to the play.

**Comment by FrancescoBorzi** [comment]
as @Kitzunu posted on Discord already:

> There is support for Chinese localization yes, but not all quests/items/etc have been translated to Chinese.
> There is no existing Portal Master module for Azerothcore, but it could probably be easily ported to AC
> You can change all loot tables in *_loot_template
> I will not suggest any repack since none is up to date, and we do not support repacks.
> And for your last question, you have this module https://github.com/azerothcore/mod-congrats-on-level
